codeserver/ubi9-python-3.9/Dockerfile

```
@@ -39,6 +39,24 @@ RUN yum install -y "https://github.com/coder/code-server/releases/download/${COD
 RUN chmod -R g+w /opt/app-root/lib/python3.9/site-packages && \
     fix-permissions /opt/app-root -P
 
+# Create a temporary directory for extensions
+WORKDIR /opt/app-root/extensions-temp
+
+COPY --chown=1001:0 utils/*.vsix /opt/vsix/
+
+# Install VS Code extensions
+RUN code-server --install-extension /opt/vsix/ms-python.python-2024.2.1.vsix --extensions-dir /opt/app-root/extensions-temp && \
```

instead of changing working directory, 
shall we do:
```suggestion
RUN mkdir -p /opt/app-root/extensions-temp && \
code-server --install-extension /opt/vsix/ms-python.python-2024.2.1.vsix --extensions-dir /opt/app-root/extensions-temp && \
```

---

Thanks for your suggestions! I'd like to further investigate them, as upon initial application, they don't seem to achieve the intended results.

---

Fixed, on my testing I used it in a non proper way. Now it works fine with your suggestion thanks!